### PR TITLE
Beta Disclaimer

### PR DESCRIPTION
**Pull request by philkra**
Heya, let's use this PR to ensure we all agree on the wording for the beta disclaimer, before I move ahead and open PR's in the repositories listed in ticket #24

### CODE REVIEW

**Review by felixbarny — APPROVED** [review]

File: beta-disclaimer.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+
+**Please note** that this is library is in a **beta** version and backwards-incompatible changes might be introduced in future releases.
```
Comment (webmat) [inline beta-disclaimer.md]: Sounds great overall. I would perhaps be more explicit that this could happen even in minor releases.

**Review by webmat — APPROVED** [review]
LGTM

I think we can be a little more explicit, but I'll let you make the call.

File: beta-disclaimer.md (hunk 1 above)
Comment (codebrain) [reply beta-disclaimer.md]: Yes, I think giving some idea of breaking in minors is better. Also, I think its important that each ecosystem is able to provide clearer directions on the versioning and compatibility, as the .NET library versioning is more subtle than this sentence suggests.

Comment (delvedor) [reply beta-disclaimer.md]: I agree with what said above, furthermore, I think this message should live in the README :)

Comment (philkra) [reply beta-disclaimer.md]: Fair point gentlemen. I added a part about possible minor BC's. Re-requesting approval

**Review by sethmlarson — APPROVED** [review]

**Pull request merged**